src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1547452866.inc.php

```diff
@@ -0,0 +1,71 @@
+<h1>Build #1547452866</h1>
+<h2>Date: 2019-01-14</h2>
+<div class="changelog">
+    - https://github.com/chameleon-system/chameleon-system/issues/265
+    - backend - main navigation: add new field icon_font_awesome
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+    ->setFields([
+        'cms_tbl_conf_id' => TCMSLogChange::GetTableId('cms_tbl_conf'),
+        'name' => 'icon_font_awesome',
```

Comment:
On second thought I'd say we should use a more neutral term for this new field. After all we can put any CSS class declarations here and don't need to restrict to Font Awesome (and we might later switch to another font), so how about simply `css_class` with the help text explaining that this field is used to display the table as menu item?

Reply:
You're right, we can use a different font in the future or add another one, but it's not a general CSS class for the navigation link, but for the icon element <i> inside the link. So I would suggest to call the field "Icon Font CSS class", respectively "icon_font_css_class".